constants.py

```diff
@@ -1,3 +1,25 @@
 UPDATE_DELAY = 86400  # 24 hours in seconds
 UPDATE_DELAY_TESTING = 60  # 1 minute in seconds
-ASSET_BASE_URL = 'https://raw.githubusercontent.com/cuappdev/assets/master/eatery/'
+CORNELL_DINING_URL = 'https://now.dining.cornell.edu/api/1.0/dining/eateries.json'
+GIT_CONTENT_URL = 'https://raw.githubusercontent.com/cuappdev'
+IMAGES_URL = GIT_CONTENT_URL + '/assets/master/eatery/eatery-images/'
+STATIC_EATERIES_URL = GIT_CONTENT_URL + '/DiningStack/master/DiningStack/externalEateries.json'
+PAY_METHODS = {
+    'brbs': 'Meal Plan - Debit',
+    'c-card': 'Cornell Card',
+    'credit': 'Major Credit Cards',
+    'mobile': 'Mobile Payments',
+    'swipes': 'Meal Plan - Swipe'
+}
+# [M, T, W, Th, F, St, S]
+# [0, 1, 2, 3, 4, 5, 6]
+WEEKDAYS = {
+    'monday-wednesday': [0, 1, 2],
+    'monday-thursday': [0, 1, 2, 3],
+    'monday-friday': [0, 1, 2, 3, 4],
+    'thursday': [3],
+    'friday': [4],
+    'saturday': [5],
+    'sunday': [6]
+}
+NUM_DAYS_STORED = 8
```

Comment:
why is this 8?

Reply:
Cornell Dining json stores today and the next week of eatery information, so our graphql db stores 8 days worth of information at any point in time

Reply:
make this more verbose, `NUM_DAYS_STORED_IN_DB`

Reply:
alphabetize this file as well